en/docs/install-and-setup/setup/deployment-best-practices/production-deployment-guidelines.md

```diff
@@ -474,6 +474,18 @@ are reflected in the below mentioned config files, according to the new configur
 <p>See <a href="{{base_path}}/administer/product-configurations/default-product-ports/">Default Product Ports</a> for
  a list of common and product-specific ports used by WSO2 products.</p>
 </div>
+<div class="panelHeader" style="border-bottom-width: 1px;">
+<strong>Firewall between Traffic Manager and Gateway</strong>
+</div>
+<p>If you have a firewall between the Traffic Manager and the Gateway, you need to configure the heartbeat value to keep the JMS connection alive. To configure this open the <code>&lt;PRODUCT_HOME&gt;/repository/conf/advanced/qpid-config.xml</code> file and set the heartbeat value to a non zero value as shown below.</p>
```

Comment:
Shouldn't &lt;PRODUCT_HOME&gt; be &lt;API-M_HOME&gt; ?

Reply:
PRODUCT_HOME is used in other places in this page and API-M_HOME is not used at all in this page. Hence used PRODUCT_HOME